tests/features/interactive_steps.feature

```diff
@@ -0,0 +1,61 @@
+@interactive
+Feature: Interactive steps
+
+# In tag names, 'i' is for 'interactive'
+
+@generated @fast @i1
+Scenario: I set text-type values
+  Given I start the interview at "all_tests"
+  And I set the "True" choice of var "checkboxes_yesno" to "true"
+  And I set the "checkbox_other_opt_1" choice of var "checkboxes_other" to "true"
+  And I set the var "dropdown_test" to "dropdown_opt_2"
+  And I set the "False" choice of var "radio_yesno" to "true"
+  And I set the "radio_other_opt_3" choice of var "radio_other" to "true"
+  And I set the var "text_input" to "Regular text input field value"
+  And I set the var "textarea" to "Multiline text\narea value"
+  When I set the "True" choice of var "direct_standard_fields" to "true"
+  # Next page
+  Then the question id should be "showifs"
```

Comment:
Is this a good place to re-test the "then I arrive at the next page" statement? I know we have it in another test, but we could test it twice since it's a trickier statement.

Reply:
It's a good thought to keep in mind, but I don't think repeating it here will serve a purpose because this doesn't trigger one of the tricky situations so we'd basically be testing the same thing. We haven't yet been able to replicate the tricky situations that have come up against. If/when we can, I think those would be useful tests. I'd appreciate your thoughts on this, though, if you have others.